### ISSUE TITLE
-Wsometimes-uninitialized in drivers/scsi/ufs/ufs-mediatek.c

### ISSUE THREAD

**Issue by nathanchance** [ClangBuiltLinux/linux#426]
```
drivers/scsi/ufs/ufs-mediatek.c:112:7: warning: variable 'ret' is used uninitialized whenever 'if' condition is false [-Wsometimes-uninitialized]
                if (on)
                    ^~
drivers/scsi/ufs/ufs-mediatek.c:120:9: note: uninitialized use occurs here
        return ret;
               ^~~
drivers/scsi/ufs/ufs-mediatek.c:112:3: note: remove the 'if' if its condition is always true
                if (on)
                ^~~~~~~
drivers/scsi/ufs/ufs-mediatek.c:108:7: warning: variable 'ret' is used uninitialized whenever 'if' condition is false [-Wsometimes-uninitialized]
                if (!on)
                    ^~~
drivers/scsi/ufs/ufs-mediatek.c:120:9: note: uninitialized use occurs here
        return ret;
               ^~~
drivers/scsi/ufs/ufs-mediatek.c:108:3: note: remove the 'if' if its condition is always true
                if (!on)
                ^~~~~~~~
drivers/scsi/ufs/ufs-mediatek.c:96:9: note: initialize the variable 'ret' to silence this warning
        int ret;
               ^
                = 0
2 warnings generated.
```

Patch sent: https://lore.kernel.org/lkml/20190320180340.25795-1-natechancellor@gmail.com/

**Comment by nathanchance** [comment]
Patch accepted: https://git.kernel.org/mkp/scsi/c/1943edacef8966e24fc66d8cbe6347fe8b10ff80

**Comment by nathanchance** [comment]
Fixed in `next-20190322`: https://git.kernel.org/next/linux-next/c/1943edacef8966e24fc66d8cbe6347fe8b10ff80
